spec/factories/users.rb

```diff
@@ -1,4 +1,20 @@
 # Read about factories at https://github.com/thoughtbot/factory_girl
+
+# It is a feature of our app that first signed up user is admin. This property
+# is set in a before create callback `setup_role` in user model.
+# We want to override this behavior when we create user factories. We want
+# `create(:user)` to create non-admin user by default. We are enforcing it
+# with `after create` blocks in following factories.
+#
+# Following commands won't work:
```

Comment:
Teah. Technically for rest two, setting `is_admin` won't work either. However, since thats their default it won't matter. Should I remove them?